ChangeLog

```diff
@@ -1,3 +1,19 @@
+3.10.4:
+	- Add a debug log for computed class in splayclass
+	- Do not ignore meta promises in server bundles (CFE-2066)
+	- Load augments at the end of context discovery
+	  This means that classes defined as part of the context discovery
+	  (e.g. 'am_policy_hub' and 'policy_server') can be used in the
+	  augments. (CFE-2482)
+	- Transform filesexist() into a collecting function (CFE-2744)
+	- Add IPv6 hard classes with the "ipv6_" prefix (CFE-2310)
+	- Don't error when calling isexecutable on broken link (CFE-741)
+	- zendesk#3739: Load process table when actually needed for a processes promise
```

Comment:
This one should have ENT-2536 instead of zendesk.

Reply:
Done! But see my question below...